common_test/src/main/java/uk/gov/dhsc/htbhf/TestConstants.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.dhsc.htbhf;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class TestConstants {
```

Comment:
I want to start moving the TestConstants out of the dwp-api module into the common-test module as they are independant of the dwp-api module really, but will do in a separate PR. These constant will then be more accessible.